Linker error with Visual Studio 2017

Hi Jason,
The compilation is Ok  but the linkage display this error !?

error LNK2001: unresolved external symbol "public: void __cdecl XPC::Timer::stop(void)" (?error LNK2001: unresolved external symbol "public: void __cdecl XPC::Timer::start
....
1>..\XPlaneConnect\64\win.xpl : fatal error LNK1120: 2 unresolved externals
1>Done building project "xpcPlugin.vcxproj" -- FAILED.

Have a nice day,

Thank you

```
1>x64\Release\Timer.obj : warning LNK4042: object specified more than once; extras ignored
```
I also see this line in the build log but it's suspicious. I noticed that the Item Type of `Timer.h` is set to `C/C++ compiler` while it should be `C/C++ header`.

Could you right click on `Timer.h` in VS,  select `Properties > General`  and change the `Item Type` to `C/C++ header`. Then clean and build again.


![xcp2](https://user-images.githubusercontent.com/707703/62491096-52636300-b7cb-11e9-953b-84906568ebf4.JPG)

--- a/xpcPlugin/xpcPlugin/xpcPlugin.vcxproj
+++ b/xpcPlugin/xpcPlugin/xpcPlugin.vcxproj
@@ -206,7 +206,7 @@
     <ClInclude Include="..\Log.h" />
     <ClInclude Include="..\Message.h" />
     <ClInclude Include="..\MessageHandlers.h" />
-    <ClCompile Include="..\Timer.h" />
+    <ClInclude Include="..\Timer.h" />
     <ClInclude Include="..\UDPSocket.h" />
   </ItemGroup>
   <ItemGroup>

